salt/states/elasticsearch.py

```diff
@@ -41,9 +41,9 @@ def index_absent(name):
                     ret['comment'] = 'Failed to remove index {0} for unknown reasons'.format(name)
         else:
             ret['comment'] = 'Index {0} is already absent'.format(name)
-    except Exception as e:
+    except Exception as err:
         ret['result'] = False
-        ret['comment'] = str(e)
+        ret['comment'] = err
```

Comment:
This should be `six.text_type(err)`, otherwise you'll get a msgpack error because the comment will contain an exception instance (which is not serializable).